### PR TITLE
UCL-352 fixed tooltip, dialog and progressbar documentation

### PR DESCRIPTION
**Pull request by Danielvandervelden**

### CODE REVIEW

File: stories/Tooltip/Tooltip.mdx [hunk 1]
```diff
@@ -5,13 +5,77 @@ import * as TooltipStories from "./Tooltip.stories.tsx";
...
+      <td>title</td>
+      <td>string</td>
+      <td>no</td>
+      <td></td>
+      <td>The title that will be displayed in the tooltip popup.</td>
+    </tr>
+    <tr>
+      <td>children</td>
+      <td>ReactNode</td>
+      <td>no</td>
```
Comment (pawel-napieracz) [inline stories/Tooltip/Tooltip.mdx:45]: it's required

File: stories/Tooltip/Tooltip.mdx [hunk 2]
```diff
@@ -5,13 +5,77 @@ import * as TooltipStories from "./Tooltip.stories.tsx";
...
+      <th>Type</th>
+      <th>Required</th>
+      <th>Default</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>label</td>
+      <td>string</td>
```
Comment (pawel-napieracz) [inline stories/Tooltip/Tooltip.mdx:28]: label: string | ReactNode;

**Review by pawel-napieracz — DISMISSED** [review]

File: stories/Tooltip/Tooltip.mdx [hunk 3]
```diff
@@ -5,13 +5,77 @@ import * as TooltipStories from "./Tooltip.stories.tsx";
...
+      <td>no</td>
```
[the remark this answers is not in the document]
Comment (Danielvandervelden) [reply stories/Tooltip/Tooltip.mdx:29]: fixed

File: stories/Tooltip/Tooltip.mdx (hunk 1 above)
Comment (Danielvandervelden) [reply stories/Tooltip/Tooltip.mdx:45]: fixed

File: stories/Tooltip/Tooltip.mdx (hunk 2 above)
Comment (Danielvandervelden) [reply stories/Tooltip/Tooltip.mdx:28]: `ReactNode` includes `string`

Comment (Danielvandervelden) [reply stories/Tooltip/Tooltip.mdx:28]: Fixed

Comment (Danielvandervelden) [reply stories/Tooltip/Tooltip.mdx:28]: Also in the component itself

File: stories/Notifications/Dialog/Dialog.mdx [hunk 4]
```diff
@@ -11,6 +11,73 @@ Dialogs appear as a modal on top of the existing content.
 When the user presses enter without tabbing through the interface, the primary button is triggered, even when it is a destructive action.
 There is no X (close) button, no event handler for the escape key and the dialog stays open when you click outside the dialog.
 
+The dialog has different properties that can be used to customize the dialog.
+
+## Dialog properties
+
+<table>
```
[the remark this answers is not in the document]
Comment (Danielvandervelden) [reply stories/Notifications/Dialog/Dialog.mdx:18]: Fixed

**Review by MichalGorskiOnegini — APPROVED** [review]

**Pull request merged**